core/src/types/capability.rs

```diff
@@ -98,7 +98,10 @@ pub struct Capability {
     pub list_with_limit: bool,
     /// If backend supports list with start after, it will be true.
     pub list_with_start_after: bool,
-
+    /// If backend support list with delimiter
+    pub list_with_delimiter: bool,
```

Comment:
delimiter could be any char and some service may only support `/`. Please use `list_with_delimiter_slash` instead.

Reply:
updated.
also in the latest commit i have added the capabilities in the `backend.rs` files. 
also noticed that `ipmfs` did not have the `list` capability but has `list` implemented, so i have added it. let me know if this is not required.

Reply:
Thanks!